packages/db/src/core/remove.spec.ts

```diff
@@ -37,3 +43,11 @@ it("should remove an entity", async () => {
   const bob = await first(userTable, { where: { id: 1 } });
   expect(bob).toBe(null);
 });
+
+it("should correctly remove an entity from a table with index", async () => {
+  await remove(userTableWithIndex, { id: 0 });
+  const alice = await first(userTableWithIndex, { where: { id: 0 } });
+  expect(alice).toBe(null);
+  const alice2 = await first(userTableWithIndex, { where: { name: "Alice" } });
+  expect(alice2).toBe(null);
```

Comment:
This expect fails without the fix (because the entry is still in the index)

Reply:
Thanks a lot for finding that bug 👍